Tests/Perf/.gitignore

```diff
@@ -0,0 +1,92 @@
+.DS_Store
```

Comment:
Was there anything missing from the root .gitignore that you needed? We could consider adding there instead of committing another .gitignore file.